src/utils.js

```diff
@@ -0,0 +1,33 @@
+import { resourceEndpoints }  from './routes/base';
+
+/*
+ * Module with utility shared methods
```

Comment:
We are using it in routes generation as well, and might be helpful in other places.